README.md

```diff
@@ -148,7 +148,7 @@ We welcome security research on DEV under the terms of our [vulnerability disclo
 
 This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. Please see the [LICENSE](./LICENSE.md) file in our repository for the full text.
 
-Like many open source projects, we require that contributors provide us with a Contributor License Agreement (CLA). By submitting code to the DEV project, you are granting us a right to use that code under the terms of the CLA.
+Like many open-source projects, we require that contributors provide us with a Contributor License Agreement (CLA). By submitting code to the DEV project, you are granting us a right to use that code under the terms of the CLA.
```

Comment:
In my experience, open source is rarely hyphenated. Personally, I like to use title case for it. If you want to change this to `Open Source` that would be good.